lib/src/sql/value/value.rs

```diff
@@ -1199,6 +1199,7 @@ impl ops::Sub for Value {
 	fn sub(self, other: Self) -> Self {
 		match (self, other) {
 			(Value::Number(v), Value::Number(w)) => Value::Number(v - w),
+			(Value::Datetime(v), Value::Datetime(w)) => Value::Duration(v - w),
```

Comment:
Regardless of whether this PR is accepted in full, this part makes a lot of sense!

Reply:
I now realize that it would probably have been way smarter of me to split this into two separate PRs but oh well 😅 

TBH, I was actually surprised when delving into the code to realize that substracting `Datetime`s wasn't supported.